### ISSUE TITLE
Should complain about missing parent for the direct child > selector

### ISSUE THREAD

**Issue by limonte** [stylelint/stylelint#4219]
> Which rule, if any, is the bug related to?

That's what I'm trying to figure out :)

> What CSS is needed to reproduce the bug?

```css
> .class-name {
  display: none;
}
```

`> .class-name` doesn't make sense becase `>` stand for direct child, but parent isn't provided.

### Reproduce

It's possible to reproduce the issue with the CSS code provided earlier in this issue-report: https://stylelint.io/demo

**Comment by evilebottnawi** [comment]
We doesn't check syntax because it is out of scope stylelint, you can write plugin for this, also it is very difficult because we support many preprocessors like sass/less and etc

**Comment by limonte** [comment]
Thank you @evilebottnawi! Do you maybe have some recommendation on css/scss validators? I'm unable to find anything which might help me to prevent such types of issues in future and any recommendation is very welcome.

**Comment by hudochenkov** [comment]
There is no such plugin yet :(
